projects/core/src/anonymous-consents/store/effects/anonymous-consents.effect.spec.ts

```diff
@@ -178,6 +179,19 @@ describe('AnonymousConsentsEffects', () => {
     });
   });
 
+  const giveRequiredConsentsMethod = 'giveRequiredConsents';
```

Comment:
Maybe we can also verify that the method was called from `loadAnonymousConsentTemplates$`?